libuavcan/include/uavcan/node/generic_subscriber.hpp

```diff
@@ -217,6 +229,16 @@ class UAVCAN_EXPORT GenericSubscriber : public GenericSubscriberBase
     }
 
     TransferListenerType* getTransferListener() { return forwarder_; }
+
+ public:
+    /**
+     * Set filter function determining if a given transfer will be decoded.
+     * By default, or if \p filter is nullptr, all messages will be decoded.
```

Comment:
It should be clarified here that if the predicate returns false, the transfer will be dropped (ignored).